notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-095_settling_time_rms.ipynb

```diff
@@ -0,0 +1,907 @@
+{
```

Comment:
<p>I assume that the 2 last cells can be removed ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>